experimenter/experimenter/nimbus-ui/src/hooks/useOutcomes.tsx

```diff
@@ -18,8 +18,19 @@ export function useOutcomes(experiment: getExperiment_experimentBySlug): {
       return [];
     }
 
+    if (experiment.application === null) {
+      return slugs
+        .map((slug) => outcomes!.find((outcome) => outcome!.slug === slug))
+        .filter((outcome) => outcome != null);
+    }
```

Comment:
Application can never be null, is this here because typescript forces you to deal with the null case?

Reply:
its because the graphql is incorrect

Reply:
https://github.com/mozilla/experimenter/blob/a518f6feda664a67c1e2a97a12a59713f371187c/experimenter/experimenter/experiments/api/v5/types.py#L335

this should be `graphene.NonNull(graphene.List(graphene.NonNull(NimbusOutcomeType))` (or something similar)

Reply:
Oops I relized i commented about outcome, not application. Thats https://github.com/mozilla/experimenter/blob/a518f6feda664a67c1e2a97a12a59713f371187c/experimenter/experimenter/experiments/api/v5/types.py#L136 and it should be `graphene.NonNull(...)`

Reply:
Updated these and regenerated types.

Reply:
Oh yeah I wasn't gonna say we should change all the types, probably just a `!` would've been sufficient.